.github/tests/forks/fork9.yml

```diff
@@ -23,3 +23,5 @@ args:
   additional_services:
     - pless_zkevm_node
     - tx_spammer
+
+  deploy_l2_contracts: true
```

Comment:
I’m considering the idea of creating a default parameter file with options that could be applied to all forks (e.g. `deploy_l2_contracts: true` and maybe others. This might be something we can explore later.